ts/components/staking/wizard/wizard_flow.tsx

```diff
@@ -284,20 +294,24 @@ const RecommendedPoolsStakeInputPane = (props: StakingInputPaneProps) => {
                     if (label === StakingPercentageValue.Fourth) {
                         setStakeAmount(formatZrx(roundedZrxBalance / 4, { removeComma: true }).formatted);
                         setSelectedLabel(StakingPercentageValue.Fourth);
+                        trackStakingAmountSelected(StakingPercentageValue.Fourth);
                     }
                     if (label === StakingPercentageValue.Half) {
                         setStakeAmount(formatZrx(roundedZrxBalance / 2, { removeComma: true }).formatted);
                         setSelectedLabel(StakingPercentageValue.Half);
+                        trackStakingAmountSelected(StakingPercentageValue.Half);
                     }
                     if (label === StakingPercentageValue.All) {
                         setStakeAmount(formatZrx(roundedZrxBalance, { removeComma: true }).formatted);
                         setSelectedLabel(StakingPercentageValue.All);
+                        trackStakingAmountSelected(StakingPercentageValue.All);
                     }
                 }}
                 onChange={(newValue: React.ChangeEvent<HTMLInputElement>) => {
                     const newAmount = newValue.target.value;
                     setStakeAmount(newAmount);
                     setSelectedLabel(undefined);
+                    trackStakingAmountSelected('custom');
```

Comment:
This is kinda wonky but I went out of my way not to track the actual amounts to make it feel less intrusive for the users.

Reply:
Sounds good!